backend/email_generator/service.py

```
@@ -0,0 +1,26 @@
+from backend.core.llm import generate_text
```

Pylint (3.3.7)

[error] No name 'generate_text' in module 'backend.core.llm'

(E0611)